dali/operators/image/remap/warp_affine.cc

```diff
@@ -23,15 +23,22 @@ DALI_SCHEMA(WarpAffine)
   .InputLayout(0, { "HWC", "DHWC" })
   .SupportVolumetric()
   .AddOptionalArg<float>("matrix",
-      R"code(Transform matrix (dst -> src).
+      R"code(Transform matrix.
 
+When the ``inverse_map`` option is set to true (default), the matrix represents a destination to source mapping. 
 With a list of values ``(M11, M12, M13, M21, M22, M23)``, this operation produces a new image
 by using the following formula::
 
   dst(x,y) = src(M11 * x + M12 * y + M13, M21 * x + M22 * y + M23)
 
-It is equivalent to OpenCV's ``warpAffine`` operation with the ``WARP_INVERSE_MAP`` flag set.)code",
+If the ``inverse_map`` option is set to false, the matrix represents a source to destination 
+transform and it is inverted before applying to the formula above.
+
+It is equivalent to OpenCV's ``warpAffine`` operation with the ``inverse_map`` argument being 
+analog to the ``WARP_INVERSE_MAP`` flag.)code",
```

Comment:
I don't know if we should use the behavior of other libraries as a reference in our documentation. But maybe it's just me being nitpicky.

Reply:
It think this line doesn't serve a purpose of defining anything in this doc, but rather be a hint for users familiar with opencv. That said, all the needed information must be still in the docs, even without this sentence.